data/constants/keycodes/keycodes_0.0.2.hjson

```diff
@@ -1,5 +1,8 @@
 {
     "ranges": {
+        "0x52E0/0x001F": {
+            "define": "QK_POINTING_MODE"
```

Comment:
It would make things simpler if this was broken up into MO and TG separately.

Reply:
Yeah I went through and changed this and removed some extra defines that are no longer needed hopefully I caught everything.